example/ios/Podfile.lock

```diff
@@ -0,0 +1,344 @@
+PODS:
```

Comment:
I guess we don't need to keep `.lock` file here, as it's example app? Just to avoid confusion, such as having different pods version.

Reply:
Included it in gitignore.